lib/bridge/MatrixHandler.js

```diff
@@ -260,6 +261,10 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
                 summary: "Issue a raw IRC command. These will not produce a reply." +
                          "(Note that the command must be all uppercase.)",
             },
+            "!bridgeversion": {
+                example: `!bridgeversion`,
+                summary: "Return the version from matrix-appservice-irc-bridge.",
```

Comment:
Minor: the dash between irc and bridge shouldn't be there.

Reply:
Resolved in additional commits